inst/test-refs/1_summary_obj.R

```diff
@@ -1,13 +1,14 @@
 structure(list(
   absolute_model_path = "/data/home/sethg/bbr/inst/model/nonmem/basic/1",
   run_details = list(
-    version = "7.4.4", run_start = "-999999999",
+    version = "7.4.4", run_start = NA_character_, #"-999999999",
```

Comment:
I'm not sure where these commented `999...` are coming from in this file, but I think you just want to regenerate this fresh with `source("data-raw/build-model-summary-refs.R")`

Reply:
ooh good catch jeez. Yeah I modified this directly and forgot it was generated